nautobot/docs/user-guide/administration/installation/nautobot.md

```diff
@@ -266,6 +266,45 @@ Nautobot does not come with any predefined user accounts. You'll need to create
 nautobot-server createsuperuser
 ```
 
+## Build Nautobot 2.0 UI
+
+Nautobot 2.0 introduces a new user interface. This interface is built using [React](https://react.dev/) and requires Node.js version 18 or higher to build the UI. 
+
+### Install Node.js
+
+Node.js version 18 is only listed on package manager repositories for Debian 12 Bookworm, Ubuntu 22.10 kinetic and RHEL 8.
```

Comment:
Wonder if we should update `user-guide/administration/installation/index.md` system requirements to explicitly recommend these as minimum versions? ("Ubuntu 20.04 or later is supported, but installation is easier on 22.10 or later"...?) Or even make those our documented requirements for 2.0?

Reply:
22.10 is not an LTS release of Ubuntu and Debian 12 was just released last month so I think we should leave our OS requirements alone for those for now. I'm not sure about RHEL. One one hand, anyone running RHEL 7 is probably capable of compiling a dependency from source. On the other hand RHEL 8 is only 4 years old which might not be new enough for some corporate environments to upgrade yet? We should probably drop the CentOS references since that project died a few years ago?